src/libraries/System.Text.RegularExpressions/gen/Resources/Strings.resx

```diff
@@ -0,0 +1,303 @@
+              <xsd:attribute name="type" type="xsd:string" />
+              <xsd:attribute name="mimetype" type="xsd:string" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="assembly">
+            <xsd:complexType>
+              <xsd:attribute name="alias" type="xsd:string" />
+              <xsd:attribute name="name" type="xsd:string" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="InvalidRegexGeneratorAttributeMessage" xml:space="preserve">
+    <value>The RegexGeneratorAttribute is malformed.</value>
+  </data>
+  <data name="MultipleRegexGeneratorAttributesMessage" xml:space="preserve">
+    <value>Multiple RegexGeneratorAttributes were applied to the same method. Only one is allowed.</value>
+  </data>
+  <data name="InvalidRegexArgumentsMessage" xml:space="preserve">
+    <value>The specified regex is invalid. '{0}'</value>
+  </data>
+  <data name="RegexMethodMustHaveValidSignatureMessage" xml:space="preserve">
+    <value>Regex method must be static, partial, parameterless, non-generic, and return Regex.</value>
+  </data>
+  <data name="InvalidLangVersionMessage" xml:space="preserve">
+    <value>C# LangVersion of 10 or greater is required.</value>
+  </data>
```

Comment:
There is an RSxxxx analyzer that should have flagged this.

https://github.com/dotnet/roslyn-analyzers/blob/a0522afd52b25ad211298e308fd5bbc7734865e0/src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx#L505-L507

https://github.com/dotnet/roslyn-analyzers/blob/a0522afd52b25ad211298e308fd5bbc7734865e0/src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx#L499-L501

Titles and messages shouldn't contain periods per the guidelines defined by that analyzer.

Reply:
This goes against the very fiber of my being :)  But ok.